src/classes/UTIL_OrgTelemetry_BATCH.cls

```diff
@@ -85,11 +86,17 @@ public without sharing class UTIL_OrgTelemetry_BATCH implements Database.Batchab
     public void finish(Database.BatchableContext info) {
         Integer cnt = Database.countQuery('SELECT Count() FROM npe03__Recurring_Donation__c LIMIT 1');
         if (cnt == 1) {
+            Database.executeBatch(new UTIL_OrgTelemetry_SObject_BATCH(
+                    UTIL_OrgTelemetry_SObject_BATCH.TelemetryBatchJobMode.RECURRING_DONATIONS_OPPS), 2000);
+
             Database.executeBatch(new UTIL_OrgTelemetry_SObject_BATCH(
```

Comment:
See previous note on the recurring donations job batch size.

Reply:
Should we have a comment (or local var and description, or property) here indicating why certain batch sizes are used so anyone changing the code down the road does not change the batch size?

Reply:
Good idea, I'll add a constant with a description